wayne-bremser/Homework-2/temp-converter/js/main.js

```diff
@@ -0,0 +1,48 @@
+/*
+    1. Create a function that accepts a temperature value in Fahrenheit and converts it to Celsius.
+      Note: Equation to convert F to C:
+      C = (F - 32) / 1.8
+      Test your function.
+    2. Create a second function that accepts a temperature value in Celsius and converts it to Fahrenheit.
+      Note: Equation to convert C to F:
+      F = 1.8 * C + 32
+      Test your function. */
+      function fahrenheitToCelsius (temp) {
+            return outputtemp = ((temp - 32) / 1.8).toFixed(1);
```

Comment:
I love your use of toFixed() here to avoid ridiculous decimals after the conversion!